bedrock/settings/base.py

```diff
@@ -260,13 +262,16 @@ def JINJA_CONFIG():
 PIPELINE_COMPILERS = (
     'pipeline.compilers.less.LessCompiler',
 )
-PIPELINE_LESS_BINARY = path('node_modules', 'less', 'bin', 'lessc')
```

Comment:
I actually ran into a situation where the default wasn't working and I had to set the path locally, which is why I left this in.